include/alpaka/acc/AccCpuOmp2Blocks.hpp

```diff
@@ -154,7 +155,7 @@ namespace alpaka
         {
             ALPAKA_FN_HOST static auto getAccName() -> std::string
             {
-                return "AccCpuOmp2Blocks<" + std::to_string(TDim::value) + "," + typeid(TIdx).name() + ">";
+                return "AccCpuOmp2Blocks<" + std::to_string(TDim::value) + "," + core::demangle<TIdx> + ">";
```

Comment:
would
```suggestion
                return core::demangle<AccCpuOmp2Blocks<TDim, TIdx>>;
```
work ?

Reply:
The question is how much control we want to have of the returned string. MSVC for example prepends `struct`/`class` to each class name. See e.g.: https://godbolt.org/z/6P3dfd1Mq.

But since `getAccName()` relied on `typeid` so far, we were already not giving stable names. For me, your proposed change is fine.

Reply:
It works but the prints are, for example:
_before_: `AccCpuSerial<3,unsigned long>`
_after_: `alpaka::AccCpuSerial<std::integral_constant<unsigned long, 3ul>, unsigned long>`

Reply:
Ah, I see, while Tony's approach would be more consistent across different architectures, right ?

Reply:
That is indeed a bit long. Maybe keep the shorter names then!